src/update_engine/delta_diff_generator.cc

```diff
@@ -1243,6 +1353,8 @@ bool DeltaDiffGenerator::GenerateDeltaUpdateFile(
 
   LOG(INFO) << "Reading files...";
 
+  vector<InstallOperation> kernel_ops;
```

Comment:
Doesn't look like it'll ever be. It just needs to be a vector to pass it to `generator.Add` but that could be hidden away with an overload that takes a single operation.